.github/workflows/docs-content-check-legacy-links-format.yml

```diff
@@ -28,17 +28,6 @@ jobs:
       REPO: ${{ inputs.repo-name }}
       COMMIT_SHA: ${{ inputs.commit-sha }}
     steps:
-      - name: Get target_url hostname
-        id: get-target-url-hostname
-        uses: actions/github-script@v6
-        env:
-          DEPLOYMENT_STATUS_TARGET_URL: ${{ github.event.deployment_status.target_url }}
-        with:
-          script: |
-            const { DEPLOYMENT_STATUS_TARGET_URL } = process.env
-            const { hostname } = new URL(DEPLOYMENT_STATUS_TARGET_URL)
-            return hostname
-
       - name: Get PR number
         id: get-pr-number
         uses: actions/github-script@v6
```

Comment:
Just a future thought, we could set a job output and use[ `if: success() ....` ](https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions)as a way to approach conditionals as well. [Example in tutorials.](https://github.com/hashicorp/tutorials/blob/main/.github/workflows/check-uuid.yml#L36)